redash/handlers/query_results.py

```diff
@@ -137,6 +139,32 @@ def make_csv_response(query_result):
         headers = {'Content-Type': "text/csv; charset=UTF-8"}
         return make_response(s.getvalue(), 200, headers)
 
+    @staticmethod
+    def make_excel_response(query_result):
+        s = cStringIO.StringIO()
+
+        query_data = json.loads(query_result.data)
+        book = xlsxwriter.Workbook(s)
+        sheet = book.add_worksheet("result")
+        logging.info(query_data['columns'])
+
+        list = []
```

Comment:
`list` is a builtin function in Python. Better to name it with a different name, maybe `columns_list`?